package.json

```
@@ -118,7 +118,7 @@
         }
     },
     "engines": {
-        "node": ">=10.x",
+        "node": ">=14.0.0 <15.0.0",
```

super nitpicky: I think there should be an added space
```suggestion
        "node": ">=14.0.0 < 15.0.0",
```

---

Actually, there isnt supposed to be one based on the example here )https://docs.npmjs.com/cli/v7/configuring-npm/package-json#engines)